src/test/java/edu/cornell/kfs/module/purap/batch/service/impl/JaggaerXMLInputFileTypeTest.java

```diff
@@ -0,0 +1,39 @@
+package edu.cornell.kfs.module.purap.batch.service.impl;
+
+import edu.cornell.kfs.module.purap.batch.JaggaerXMLInputFileType;
+import org.junit.Test;
+import org.junit.jupiter.api.BeforeEach;
+import org.kuali.kfs.sys.ConfigureContext;
+import org.kuali.kfs.sys.context.KualiIntegTestBase;
+import org.kuali.kfs.sys.context.SpringContext;
+
+@ConfigureContext
+public class JaggaerXMLInputFileTypeTest extends KualiIntegTestBase {
```

Comment:
To avoid having to load up the entire KFS spring context, you could having this test class extend SpringEnabledMicroTestBase.  You should just need to add an @LoadSpringFile annotation to JaggaerXMLInputFileTypeTest and have the annotation point to an XML file that just has enough for this testing class needs.